keyboards/fatotesa/info.json

```
@@ -0,0 +1,156 @@
+{
+    "development_board": "promicro",
+    "manufacturer": "fatotesa",
+    "keyboard_name": "fatotesa",
+    "maintainer": "stefangluszek",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "caps_word": true
+    },
+    "encoder": {
+        "rotary": []
```

This'll be fine knowing you only have an encoder on the right half. The right half specific settings are more so for if the right half encoder is using different pins.
```suggestion
        "rotary": [
            { "pin_a": "D1", "pin_b": "D0" }
        ]
```